pointerevents/pointerevent_constructor.html

```diff
@@ -30,6 +30,8 @@
             var testHeight = 5;
             var testTiltX = -45;
             var testTiltY = 30;
+            var testButton = 0;
+            var testButtons = 1;
```

Comment:
I also think there's a difference between "they technically can because an implementation doesn't validate" and "the spec requires that nonsense be passed-through unmolested."  

I'm not asking that the spec require validation in the constructor, but I do think this PR should be merged.  If we wanna specify that `pressure` is always passed-through in the constructor, even if `buttons` is unset, that should be a separate conversation (and a separate test).